bin/utils.js

```diff
@@ -0,0 +1,22 @@
+var fs = require('fs-extra');
+var path = require('path');
+var resolve = path.resolve;
+
+/**
+ * Gather configuration object from file.
+ * In case of a .js file the config will be required as a module.
+ * Otherwise an attempt to parse the file as JSON is being made.
+ *
+ * @param {String} configFile
+ * @returns {Object} The config object.
+ */
+exports.requireOrParse = function(configFile) {
+  var configPath = resolve(configFile);
+  var ext = path.extname(configPath);
+
+  if (/js/.test(ext)) {
```

Comment:
nitpick: we could avoid the regexp here with `ext.indexOf('js') === 0` otherwise it should be `^js(?:on)?$`